features/car/route_relations.feature

```diff
@@ -184,6 +248,11 @@ Feature: Car - route relations
             | a,b       | eastbound,eastbound  | I 80 $east,I 80 $east |
 
     Scenario: Direction only available via superrelation direction
+        Given the profile file "car" initialized with
+        """
+        profile.cardinal_directions = true
```

Comment:
for completeness, I would like to see a test with explicitly setting `profile.cardinal_directions = false`